app/views/course/assessment/assessments/show.html.slim

```diff
@@ -31,15 +31,17 @@
       - if current_course.gamified?
         tr.bonus_cut_off
           th = t('.bonus_cut_off')
-          - if @assessment.bonus_end_at.present?
-            td = render partial: 'course/lesson_plan/items/personal_or_ref_time',
-                        locals: { item: @assessment, course_user: current_course_user, attribute: :bonus_end_at,
-                                  datetime_format: :long }
+          td
+            - if @assessment.bonus_end_at.present?
+              = render partial: 'course/lesson_plan/items/personal_or_ref_time',
+                          locals: { item: @assessment, course_user: current_course_user, attribute: :bonus_end_at,
```

Comment:
Can align "locals" with "partial" on the previous line, not sure why Hound is feeling charitable today 😂